src/jit/compphases.h

```diff
@@ -92,6 +92,7 @@ CompPhaseNameMacro(PHASE_RA_ASSIGN_VARS,         "RA assign vars",
 CompPhaseNameMacro(PHASE_LOWERING_DECOMP,        "Lowering decomposition",         "LWR-DEC",  false, -1, false)
 CompPhaseNameMacro(PHASE_LOWERING,               "Lowering nodeinfo",              "LWR-INFO", false, -1, true)
 #ifndef LEGACY_BACKEND
+CompPhaseNameMacro(PHASE_STACK_LEVEL_SETTER,     "Setting stack levels for x86",   "STK-SET",  false, -1, false)
```

Comment:
It's weird that this says "for x86" even though the phase runs for all platforms.